src/Stratis.SmartContracts.ByteHelper/ByteConverter.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Text;
+
+namespace Stratis.SmartContracts.ByteHelper
```

Comment:
Could we perhaps pick a name without the word 'helper' in it? I think @majikandy might have some thoughts around this.